datasets/ONS-UK-trade-in-services/main.py

```diff
@@ -1,58 +1,196 @@
 #!/usr/bin/env python
 # coding: utf-8
 # %%
+from gssutils import *
+import json
+
+cubes = Cubes("info.json")
+scraper_1 = Scraper('https://www.ons.gov.uk/businessindustryandtrade/' + \
+                  'internationaltrade/datasets/uktradeinservicesallcountriesnonseasonallyadjusted')
+scraper_1
+trace = TransformTrace()
+
+# %%
+info = json.load(open("info.json"))
+scraper_1.dataset.family = info["families"]
+
+# %%
+dist_1 = scraper_1.distributions[0]
+dist_1
+
+# %%
+first_url = dist_1.downloadURL
+first_url
+
+# %%
+tabs = {tab.name: tab for tab in scraper_1.distribution(latest=True, mediaType=Excel).as_databaker()}
```

Comment:
Above code could be refactored slightly & some legacy code removed. As there are two dataset URLs within this it would be better if we directly change within the info.json bypassing in the URL. 

Replace lines (7-26) above with two following snippets of code. 


```python
#Landing page URL in info.json for UK trade in services: all countries, non-seasonally adjusted
with open("info.json", "r") as read_file:
    data = json.load(read_file)
    data["landingPage"] = "https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/uktradeinservicesallcountriesnonseasonallyadjusted" 

with open("info.json", "w") as jsonFile:
    json.dump(data, jsonFile)
```
```python
scraper = Scraper(seed="info.json")
distribution = scraper.distribution(latest=True)

trace = TransformTrace()
cubes = Cubes("info.json")

tabs = {tab.name: tab for tab in distribution.as_databaker()}
```
Note
 - scraper = Scraper(seed="info.json")
it'll pick up the URL and family information from the info.json automatically.

- distribution = scraper.distribution(latest=True)
using latest = True instead of selecting distributions[0]

Reply:
Serializing and Deserializing of JSON included.Other codes making use of JSON file is  included.